index.ipynb

```diff
@@ -52,11 +52,9 @@
     "**In this cell and the next one, each `freud` module is linked to the documentation for more information, while the links in the list point to Jupyter notebooks demonstrating the classes in those modules.**\n",
```

Comment:
<p>The indentation of <code>freud.locality</code> is off here, but it might just be an artifact of ReviewNB. Worth a quick check though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>